http/http_client.go

```diff
@@ -17,6 +17,13 @@ type QueryParam struct {
 	value string
 }
 
+func NewQueryParam(key string, value string) QueryParam {
+	return QueryParam{
+		key:   key,
+		value: value,
+	}
+}
+
```

Comment:
why do we need this new method and why can't we use existing `Pair` method on queryParam?

something like:
```go
queryParam := http.QueryParam{}
queryParam.Pair("PR", 8970)
```

Reply:
The problem comes when you want to use it inline. 
The http method is defined as `(ctx context.Context, path string, queryParams ...QueryParam)` 
So if you want to for example: `HttpClient.Get(api.Ctx, "api", http.NewQueryParam("ps", "1"), http.NewQueryParam("p", "100"))`
Without this method it would look something like: 

```
var queryParams []http.QueryParam
queryParam1 := http.QueryParam{}
queryParam.Pair("PR", 8970)
queryParam2 = http.QueryParam{} 
queryParam2.Pair("PR", 8970)

HttpClient.Get(api.Ctx, "api", queryParam1, queryParam2)
```

For me the whole .Pair method does not really makes sense, QueryParam fields are declared as private to restrict mutations and then a method exposed to allow mutations, but I left it so it does not break other plugin uses of the plugin.

Reply:
Add some validation like this in that. case for value string.

https://github.com/digital-ai/release-deploy-integration/blob/7beb8e9e277cc098f40876067e0be3a318a15d99/deploy/cmd/query_request/query_request.go#L20

Reply:
or let's leave it like this